packages/calcite-components/src/components/tile-group/tile-group.tsx

```diff
@@ -53,7 +53,10 @@ export class TileGroup implements InteractiveComponent, SelectableGroupComponent
    *
    * Use `"horizontal"` for rows, and `"vertical"` for a single column.
    */
-  @Prop({ reflect: true }) layout: Exclude<Layout, "grid"> = "horizontal";
+  @Prop({ reflect: true }) layout: Exclude<
```

Comment:
Based on this PR, I think it's better if we change this to `Extract` instead. Applies to other relevant exclusions.